kytos/core/controller.py

```diff
@@ -169,6 +170,18 @@ def register_rest_endpoint(self, *options, **kwargs):
         """Method used to return the endpoints registered by APIServer."""
         self.api_server.register_rest_endpoint(*options, **kwargs)
 
+    def restart(self, graceful=True):
+        """Restart Kytos SDN Platform.
+
+        Args:
+            graceful(bool): Represents the way that Kytos will restart.
+        """
+        if self.started_at is not None:
+            self.stop(graceful)
```

Comment:
The method `stop_controller` will wait all threads finish before finish the method.